data/AlokMahapatra26.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Alok Mahapatra",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Tech Enthusiast | Web Developer | UIUX ",
+  "avatar": "https://github.com/AlokMahapatra26.png",
+  "tags": [
+    "Javascript",
+    "UI UX",
+    "MERN Stack"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/aloktwts" },
+    { "icon": "FaGithub", "url": "https://github.com/AlokMahapatra26" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/@alokmahapatra1447" }
+  ],
+  
+  
+  "links": [
+   
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/aloktwts",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
```

Comment:
Hmm. Not sure you can use color fields in the links. 🤔

Reply:
You can, idk the specifics, but colors can be set on links.